specification/storage/data-plane/Microsoft.StorageDataLake/stable/2019-12-12/DataLakeStorage.json

```diff
@@ -472,6 +472,18 @@
               "x-ms-continuation": {
                 "description": "If the number of paths to be listed exceeds the maxResults limit, a continuation token is returned in this response header.  When a continuation token is returned in the response, it must be specified in a subsequent invocation of the list operation to continue listing the paths.",
                 "type": "string"
+              },
+              "x-ms-server-encrypted: true/false": {
+                "description": "The value of this header is set to true if the contents of the request are successfully encrypted using the specified algorithm.",
+                "type": "boolean"
+              },
+              "x-ms-encryption-key-sha256": {
```

Comment:
I don't think this property is in the right place.  Isn't this a per-Path property?